packages/ui/src/lib/progress/Spinner.svelte

```diff
@@ -1,13 +1,18 @@
 <script lang="ts">
-export let size = '2em';
+interface Props {
+  size?: string;
+  class?: string;
+  style?: string;
+}
+let { size = '2em', class: classProp, style: styleProp }: Props = $props();
```

Comment:
```suggestion
let { size = '2em', class: className, style: styleProp }: Props = $props();
```

We seems to started using className is other places, might need to add it to CODE_GUIDELINES, or discuss if there is different opinion

Reply:
Do we need to rename style? class is a JS keyword so make sense, but slyle is okay?

Reply:
Both good points, I've renamed to className and removed the style renaming.